src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -46,6 +46,9 @@ public static class LanguageServerConstants
 
         public const string RazorServerReadyEndpoint = "razor/serverReady";
 
+        // Same as in Web Tools
+        public const string RazorWrapWithTagEndpoint = "textDocument/_vsweb_wrapWithTag";
```

Comment:
Could we leave a big comment here indicating where this is coming from / expected flows by relying on this?